server/service/src/dashboard/item_count.rs

```diff
@@ -146,16 +146,19 @@ mod item_count_service_test {
                             id: "listline1".to_string(),
                             item_link_id: "item1".to_string(),
                             master_list_id: "list1".to_string(),
+                            ..Default::default()
                         },
                         MasterListLineRow {
                             id: "listline2".to_string(),
                             item_link_id: "item2".to_string(),
                             master_list_id: "list1".to_string(),
+                            ..Default::default()
                         },
                         MasterListLineRow {
                             id: "listline3".to_string(),
                             item_link_id: "item3".to_string(),
                             master_list_id: "list1".to_string(),
+                            ..Default::default()
```

Comment:
thanks for adding these defaults, btw i had a scripts that does something like this for the whole file, but for inline_init method, not sure if that's helpful since the ..Default::default() syntax is easier to understand and more common.

Reply:
Yeah it's a pain, but hopefully a one time thing now that we've done it.
Was considering a script or AI support option for next time :)